main/sd/source/ui/slideshow/slideshowimpl.cxx

```diff
@@ -236,6 +236,17 @@ AnimationSlideController::AnimationSlideController( Reference< XIndexAccess > xS
 		mnSlideCount = xSlides->getCount();
 }
 
+void AnimationSlideController::setStartSlideNumber( sal_Int32 nSlideNumber )
+{
+	while( ( !maSlideVisible[nSlideNumber] ) 
+			|| ( nSlideNumber > maSlideVisible.size() ) ) nSlideNumber++;
+
+	if(nSlideNumber > maSlideVisible.size())
+		mnStartSlideNumber = 0;
```

Comment:
Should all these checks: `(nSlideNumber > maSlideVisible.size())` be `>=` instead of `>`?

Reply:
I don't know because I have no idea what maSlideVisible.size is supposed to return.

Reply:
This is a draft now until the code was successfully compiled by someone.